tools/ncf.py

```diff
@@ -547,7 +547,11 @@ def generate_technique_content(technique, methods):
     else:
       promiser = "method_call"
 
-    content.append('    "'+promiser+'" usebundle => '+method_call['method_name']+'('+arg_value+'),')
+    class_parameter_id    = method_info["class_parameter_id"]
+    class_parameter_name  = method_info["parameter"][class_parameter_id]["name"]
+    class_parameter_value = method_call["args"][class_parameter_id]
+    content.append('    "'+promiser+'_context" usebundle => current_technique_report_info("'+method_info["name"]+'", "'+class_parameter_name+'", "'+class_parameter_value+'");')
```

Comment:
is it properly escaped ? what if it component key contains " or ' ?

Reply:
This doesn't match current_technique_report_info call, the first parameter is technique_name

Reply:
Thanks @ncharles ! this needs escaping you are right :( :(

Reply:
Thanks @peckpeck  it should be the technique name not the method name ...